src/main/webapp/app/exercises/programming/manage/grading/programming-exercise-grading-table-actions.component.ts

```diff
@@ -8,28 +13,25 @@ import { ProgrammingExercise } from 'app/entities/programming-exercise.model';
  */
 @Component({
     selector: 'jhi-programming-exercise-grading-table-actions',
-    template: `
-        <button
-            id="save-table-button"
-            class="btn btn-primary ms-3 my-1"
-            jhiTranslate="artemisApp.programmingExercise.configureGrading.save"
-            (click)="onSave.emit()"
-            [disabled]="isSaving || !hasUnsavedChanges"
-        ></button>
-        <button
-            id="reset-table-button"
-            class="btn btn-secondary ms-3 my-1"
-            (click)="onReset.emit()"
-            [disabled]="isSaving"
-            jhiTranslate="artemisApp.programmingExercise.configureGrading.reset"
-        ></button>
-    `,
+    templateUrl: './programming-exercise-grading-table-actions.component.html',
 })
 export class ProgrammingExerciseGradingTableActionsComponent {
+    readonly faCopy = faCopy;
```

Comment:
We normally have the icons below all other attributes and for whatever reason not as readonly

Reply:
this code is not part of my PR, so I'm not sure if I should change anything here.